apps/core/src/hooks/useTransactionSummary.ts

```diff
@@ -24,9 +24,11 @@ import { getObjectDisplayLookup } from '../utils/transaction/getObjectDisplayLoo
 export function useTransactionSummary({
 	transaction,
 	currentAddress,
+	recognizedPackagesList,
```

Comment:
Passing recognizedPackagesList as a prop, because useRecognizedPackages is different for to the explorer or wallet. I will look into combining these two prop